spec/unit/installer/xcode/target_validator_spec.rb

```diff
@@ -519,6 +520,29 @@ def create_validator(sandbox, podfile, lockfile)
             @validator.pod_targets.find { |pt| pt.name == 'matryoshka' }.stubs(:should_build?).returns(false)
             lambda { @validator.validate! }.should.not.raise
           end
+
+          it 'does not raise when a pre built swift target depends upon a target that requires building' do
+            fixture_path = ROOT + 'spec/fixtures'
+            config.repos_dir = fixture_path + 'spec-repos'
+            podfile = Podfile.new do
+              project(fixture_path + 'SampleProject/SampleProject').to_s
+              platform :ios, '12.0'
+              install! 'cocoapods', :integrate_targets => false
+              pod 'VendoredSwiftFramework', :path => (fixture_path + 'prebuilt-swift-framework').to_s
```

Comment:
this is a nice fixture of a pre-built swift framework that cocoapods 1.11.x now treats `uses_swift?` as `true`.